app.js

```diff
@@ -76,17 +75,36 @@ class AwesomeBooks {
     });
   }
 
-  reloadPage() {
-    window.onload = () => {
-      if (localStorage.getItem('books') !== null) {
-        this.bookArray = JSON.parse(localStorage.getItem('books'));
-        this.populateTable();
-      }
-    };
-  }
+insertDateTime = () => {
+  // eslint-disable-next-line
+  const DateTime = luxon.DateTime.now();
+  const timeDisplay = document.getElementById('time-display');
+  timeDisplay.innerHTML = DateTime;
+};
+
+reloadPage() {
+  window.onload = () => {
+    if (localStorage.getItem('books') !== null) {
+      this.bookArray = JSON.parse(localStorage.getItem('books'));
+      this.populateTable();
+      this.insertDateTime();
+    }
+  };
+}
 }
```

Comment:
- [ ] You are supposed to add a date below the navigation bar and you should use [Luxon]() library

Reply:
Please take a look at the screenshot, I have already added the luxon clock it working on my local machine, but it's not showing up on the GitHub build, same problem on Netlify, can you help?